content/en/docs/armory-agent/agent-mtls.md

```diff
@@ -0,0 +1,466 @@
+              security:
+                enabled: true
+                certificateChain: file:///opt/clouddriver/cert/tls.crt
+                # list of crts
+                trustCertCollection: file:///opt/clouddriver/cert/tls.crt
+                # cacert
+                privateKey: file:///opt/clouddriver/cert/tls.key
+                clientAuth: REQUIRE
+{{< /prism >}}
+
+See the {{< linkWithTitle "agent-plugin-options.md" >}} page for additional options.
+
+## Agent service configuration
+
+### Create a secret
+
+Create a secret in the target cluster namespace where the Agent resides.
+
+```bash
+kubectl create secret generic <agent-secret-name> \
+--from-file=agent.crt --from-file=agent.key
+```
+
+### Modify deployment configuration
+
+Modify the Agent's deployment configuration to mount the certs.
+
+If you have a custom CA, you need to mount the cert into the known trusted cert location:  `/etc/ssl/cert.pem`. Golang apps in Alpine use `/etc/ssl/cert.pem` as the source of a trusted CA.
+
+  1. Obtain a copy of the `cert.pem` file from a running Agent.
+  2. Append your custom CA onto the copied `cert.pem`  (for example `cat ca.crt >> cert.pem`)
+  3. Mount `cert.pem` to Agent deployment
+
+In the following example, see lines 53-54 and 74-76 for how to mount the certs and lines 59-60 and 80-82 for how to mount the custom certs.
+
+{{< prism lang="yaml" line="53-54, 59-60, 74-76, 80-82" >}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: spin
+    app.kubernetes.io/name: kubesvc
+    app.kubernetes.io/part-of: spinnaker
+    cluster: spin-kubesvc
+  name: spin-kubesvc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spin
+      cluster: spin-kubesvc
+  template:
+    metadata:
+      labels:
+        app: spin
+        app.kubernetes.io/name: kubesvc
+        app.kubernetes.io/part-of: spinnaker
+        cluster: spin-kubesvc
+    spec:
+      # imagePullSecrets:
+      # - name: regcred
+      containers:
+      - env:
+        - name: GRPC_GO_LOG_SEVERITY_LEVEL
+          value: INFO
+        - name: GRPC_GO_LOG_VERBOSITY_LEVEL
+          value: "9999"
+        image: armory/kubesvc
+        imagePullPolicy: IfNotPresent
+        name: kubesvc
+        ports:
+          - name: health
+            containerPort: 8082
+            protocol: TCP
+          - name: metrics
+            containerPort: 8008
+            protocol: TCP
+        readinessProbe:
+          httpGet:
+            port: health
+            path: /health
+          failureThreshold: 3
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /opt/kubesvc/cert
+          name: armoryagentcert
+        - mountPath: /opt/spinnaker/config
+          name: volume-kubesvc-config
+        - mountPath: /opt/kubesvc/cacert  # this didn't work as of Nov 2020
```

Comment:
Can we exclude this config or fix it if it doesn't work? @th3morg knows how to configure mTLS